sdks/node.md

```diff
@@ -77,75 +80,67 @@ The NodeJS SDK(s) are available via [NPM](https://www.npmjs.com/search?q=%40band
 ```bash
 npm install @bandwidth/messaging
 npm install @bandwidth/voice
-npm install @bandwidth/bxml
 ```
 
-## Initialize Bandwidth Client
+## Create Phone Call
 
 ```js
-const BandwidthMessaging = require('@bandwidth/messaging');
-BandwidthMessaging.Configuration.basicAuthUserName = "token";
-BandwidthMessaging.Configuration.basicAuthPassword = "secret";
-const messagingController = BandwidthMessaging.APIController;
+import { Client, ApiController } from '@bandwidth/voice';
 
-const BandwidthVoice = require('@bandwidth/voice');
-BandwidthVoice.Configuration.basicAuthUserName = "username";
-BandwidthVoice.Configuration.basicAuthPassword = "password";
-const voiceController = BandwidthVoice.APIController;
-
-const BandwidthBxml = require('@bandwidth/bxml');
-```
+const client = new Client({
+    basicAuthPassword: 'password',
+    basicAuthUserName: 'username'
+});
 
-## Create Phone Call
+const controller = new ApiController(client);
 
-```js
-var accountId = '1234';
-
-var body = new BandwidthVoice.ApiCreateCallRequest({
-    "from"          : "+19999999999",
-    "to"            : "+18888888888",
-    "applicationId" : "123",
-    "answerUrl"     : "https://test.com",
-    "answerMethod"  : "POST",
-    "callTimeout"   : 30
+const response = await controller.createCall('account-id-123', {
+    applicationId: 'app-id-123',
+    from: '+19999999999',
+    to: '+18888888888',
+    answerUrl: 'https://your-server.com/webhooks/answer',
+    answerMethod: 'POST',
+    callTimeout: 30
 });
-var response = await voiceController.createCall(accountId, body);
+
 console.log(response);
 ```
 
 ## Generate BXML
 
 ```js
-var speakSentence = new BandwidthBxml.Verbs.SpeakSentence();
-speakSentence.setSentence("test");
-speakSentence.setVoice("susan");
-speakSentence.setGender("female");
-speakSentence.setLocale("en_US");
+import { SpeakSentence, Response } from '@bandwidth/voice';
 
-var response = new BandwidthBxml.Response();
-response.addVerb(speakSentence);
+const speakSentence = new SpeakSentence({
+    sentence: 'This is a spoken test.',
+    locale: 'en_US'
+    voice: 'susan',
+    gender: 'female'
+});
+
+const response = new Response(speakSentence);
 
 console.log(response.toBxml());
 ```
 
 ## Send Text Message
 
 ```js
-var accountId = '1234';
+import { Client, ApiController } from '@bandwidth/messaging';
 
-var body = new BandwidthMessaging.MessageRequest({
-    "applicationId" : applicationId ,
-    "to"            : ["+19999999999"],
-    "from"          : "+18888888888",
-    "text"          : "The quick brown fox jumps over a lazy dog."
+const client = new Client({
+    basicAuthPassword: 'password',
+    basicAuthUserName: 'username'
 });
 
-var response = await messagingController.createMessage(accountId, body);
-console.log(response);
-```
+const controller = new ApiController(client);
 
-## Order Phone Number
+const response = await controller.createMessage('user-id-123', {
```

Comment:
user-id is confusing - this should be accountId - customers get tripped up and put their api username in that field